app/code/Magento/Catalog/Ui/DataProvider/Product/Form/Modifier/Eav.php

```diff
@@ -777,11 +777,14 @@ public function setupAttributeMeta(ProductAttributeInterface $attribute, $groupC
     private function getAttributeDefaultValue(ProductAttributeInterface $attribute)
     {
         if ($attribute->getAttributeCode() === 'page_layout') {
-            $defaultValue = $this->scopeConfig->getValue(
-                'web/default_layouts/default_product_layout',
-                \Magento\Store\Model\ScopeInterface::SCOPE_STORE,
-                $this->storeManager->getStore()
-            );
+            $defaultValue = $attribute->getDefaultValue();
+            if (!$defaultValue) {
+                $defaultValue = $this->scopeConfig->getValue(
+                    'web/default_layouts/default_product_layout',
+                    \Magento\Store\Model\ScopeInterface::SCOPE_STORE,
+                    $this->storeManager->getStore()
+                );
+            }
             $attribute->setDefaultValue($defaultValue);
```

Comment:
We don't need to update the value which is already set.
I guess the `$attribute->setDefaultValue($defaultValue);` can be also moved under `if` section.